app/components/RaceResult.tsx

```
@@ -0,0 +1,46 @@
+ * 出馬表・予想結果表示用コンポーネント
+ * @param data APIから取得したテキスト
+ */
+export function RaceResult({ data }: { data: { raceText: string; predictionText: string } }) {
+  return (
+    <section className="w-full max-w-2xl mx-auto mt-8">
+      <div className="mb-8 p-4 border rounded bg-gray-50">
+        <h3 className="font-bold mb-2">出馬表</h3>
+        <div className="prose prose-sm dark:prose-invert" dangerouslySetInnerHTML={{ __html: data.raceText }} />
+      </div>
+      <div className="p-4 border rounded bg-white">
+        <h3 className="font-bold mb-2">予想結果</h3>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)